includes/CMB2_hookup.php

```diff
@@ -356,22 +356,20 @@ public function add_context_metaboxes() {
 	/**
 	 * The opening markup for when we include a title.
 	 *
-	 * @param object  $cmb  This CMB2 object
-	 *
 	 * @return string
 	 */
-	public function context_box_title_markup_open( $cmb ) {
+	public function context_box_title_markup_open() {
 
-		$title = $cmb->prop( 'title' );
+		$title = $this->cmb->prop( 'title' );
 
 		$screen = get_current_screen();
 		$hidden = get_hidden_meta_boxes( $screen );
 
-		$classes = 'postbox context-' . $cmb->prop( 'context' ) . '-box context-box ';
-		$classes .= in_array( $cmb->cmb_id, $hidden ) ? ' hide-if-js' : '';
-		$classes .= postbox_classes( $cmb->cmb_id, $screen->id );
+		$classes = 'postbox context-' . $this->cmb->prop( 'context' ) . '-box context-box postbox-container ';
```

Comment:
The `postbox-container` class was being used to style the field rows, but I refactored it a bit so that it's not needed. You can see those changes in [Update context metabox output to handle no-title or no-wrap better](https://github.com/WebDevStudios/CMB2/pull/838/commits/c86b8bc0ba7c2cd020015c809fce5c76f5a1288d)